src/modal-template.ts

```diff
@@ -0,0 +1,187 @@
+import {
+  LitElement,
+  html,
+  css,
+  customElement,
+  CSSResult,
+  property,
+  TemplateResult,
+} from 'lit-element';
+
+import '@internetarchive/ia-activity-indicator/ia-activity-indicator';
+
+import { ModalConfig } from './modal-config';
+
+import closeIcon from './assets/close-icon';
+
+@customElement('modal-template')
+export class ModalTemplate extends LitElement {
+  @property({ type: Object }) config: ModalConfig = new ModalConfig();
+
+  render(): TemplateResult {
+    return html`
+    <div class="modal-wrapper">
+      <div class="head-spacer"></div>
+      <div class="modal-container">
+        <header style="background-color: ${this.config.headerColor}">
+          ${this.config.allowUserToClose ? this.closeButtonTemplate : ''}
+          <div class="logo-icon">
+            <img src="https://archive.org/images/logo_arches.png">
+          </div>
+          <h1>${this.config.title}</h1>
+          <h2>${this.config.subtitle}</h2>
+        </header>
+        <div class="body">
```

Comment:
**Suggestion**: rename this class name to avoid using the same name as a native HTML element.

Reply:
do we want to go further and semantically change the dom node? section, main, perhaps? - to pair with the `<header>` tag above.

Reply:
I switched this to use `<section>`